data/SocialIQA/dataset/dev-labels_small.lst

```diff
@@ -0,0 +1,10 @@
+3

```

Comment:
This hierarchy works pretty well, I think. I probably should have said `data/extracted/<benchmark>` instead of `data/<benchmark>`, but you can leave it for now.

Reply:
You do have an naming inconsistency in `submissions/` vs. `dataset/`. I'd prefer singular directory names (`submission/`).